htdocs/adherents/list.php

```diff
@@ -464,7 +464,7 @@
 }
 
 // Complete request and execute it with limit
-$sql .= $db->order($sortfield, $sortorder);
+$sql .= $db->order($sortfield, $sortorder, $sortfield == "d.ref");
```

Comment:
You mean 
$sortfield = "d.ref"
?

Reply:
No, we're passing a boolean as a parameter of a function, that is true only if $sortfield == "d.ref", false otherwise.